simpleprophet/simpleprophet/output.py

```diff
@@ -63,6 +62,23 @@ def prepare_records(modelDate, forecast_end, data, product):
     models = setup_models(years)
     forecast_start = modelDate + timedelta(days=1)
     forecast_period = pd.DataFrame({'ds': pd.date_range(forecast_start, forecast_end)})
+    actuals_data = {
+        "asofdate": modelDate,
+        "datasource": product,
+        "date": data.ds,
```

Comment:
Does our code ensure that `data` includes dates only less than `asofdate`? We should make sure to always have a continuous range of dates where the row with `asofdate = date` is the last day with `type = 'actual'` and the first date with type `forecast` is `asofdate + 1`.

Reply:
I think I fixed this.  You want to confirm it looks OK?

Reply:
Looks good.